Dockerfile

```diff
@@ -1,5 +1,5 @@
 ARG REPO=alpine
-ARG IMAGE=3.13.5@sha256:def822f9851ca422481ec6fee59a9966f12b351c62ccb9aca841526ffaa9f748
+ARG IMAGE=3.14.0@sha256:1775bebec23e1f3ce486989bfc9ff3c4e951690df84aa9f926497d82f2ffca9d
 FROM ${REPO}:${IMAGE} AS base
```

Comment:
I think it might be the usage of `ARG` that causes dependabot to not recognize the bump. If the tag was used directly in the `FROM` part, it might have detected the outdated version. Just a guess.

Reply:
That's probably right. 

Unfortunately it was necessary to write it like this to satisfy `hadolint` (I'll check they've improved that).